README.rst

```diff
@@ -32,41 +32,44 @@ Quick Start
 
 With a few steps, you can start in your client repository using CI confiurations stored in `industrial_ci`.
 
-1. Activate CI on your github repository.
+For Travis CI
+--------------
 
-- For `Travis CI <https://travis-ci.org/>`_), you may do so either at https://travis-ci.org/profile/YOUR_GITHUB_ORGANIZATION or at https://travis-ci.org/profile/YOUR_GITHUB_USER (depending on where your repository sits).
+1. Activate CI for your github repository on `Travis CI <https://travis-ci.org/>`_). You may do so either at https://travis-ci.org/profile/YOUR_GITHUB_ORGANIZATION or at https://travis-ci.org/profile/YOUR_GITHUB_USER (depending on where your repository sits).
 
-2. In `.travis.yml` file in your client repo, add in "`install`" section a sentence `git clone https://github.com/ros-industrial/industrial_ci.git .ci_config`, like below:
+2. In `.travis.yml` file in your client repo, add in "`install`" section a sentence `git clone https://github.com/ros-industrial/industrial_ci.git .industrial_ci`, like below:
 
 ::
 
   install:
-    - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config
+    - git clone https://github.com/ros-industrial/industrial_ci.git .industrial_ci
   script:
-    - .ci_config/travis.sh
+    - .industrial_ci/travis.sh
 
-* Note: The name `.ci_config` is NO longer REQUIRED for the cloned folder starting version 0.3.2; you can pick any name as long as the folder is hidden (by being prepended by ".").
+* Note: The name `.industrial_ci` is NO longer REQUIRED for the cloned folder starting version 0.3.2; you can pick any name (recommended practice to keep the folder hidden (by prepending ".").
 
-Concrete examples of config files
--------------------------------------
-
-For most of the cases, 
-For development branch intended for ROS Indigo:
-
-- `ros_canopen/.travis.yml <https://github.com/ros-industrial/ros_canopen/blob/0a42bf181804167834b8dc3b80bfca971f24546f/.travis.yml>`_
-
-For development branch intended for ROS Indigo and newer distros:
+For Gitlab CI
+-------------
 
-- `industrial_core/.travis.yml <https://github.com/ros-industrial/industrial_core/blob/eeb6a470e05233d0efaaf8c32a9e4133cdcbb80b/.travis.yml>`_: Indigo and Jade compatible.
-- `leap_motion/.travis.yml <https://github.com/ros-drivers/leap_motion/blob/954924befd2a6755f9d310f4a8b57aa526056a80/.travis.yml>`_: Indigo, Jade, Kinetic compatible. Also runs `ROS Prerelease Test <http://wiki.ros.org/bloom/Tutorials/PrereleaseTest>`_.
+1. In `.gitlab-ci.yml` file in your client repo, add in "`install`" section a sentence `git clone https://github.com/ros-industrial/industrial_ci.git .industrial_ci`, like below:
```

Comment:
You also need to _enable_ CI for your repo by assigning a suitable _runner_ to it, which can be done in _Settings > CI/CD > Runners settings_.

There is probably more than one way to set up the runner. The `.gitlab-ci.yml` in this repo is set up to use a runner configured with the Docker _executor_ (i.e. implicit by its use of `image` and `services` keywords). I do, however, think that step-by-step instructions on how to set this up are probably beyond the scope of this README.

Also, `.gitlab-ci.yml` should use `before_script` instead of `install` for cloning `industrial_ci`, and the following line should be a minimal job configuration.

I would propose something like:

1. Enable a runner for your Gitlab project which uses the Docker executor. See instructions on how to [install](https://docs.gitlab.com/runner/install/index.html) and [register](https://docs.gitlab.com/runner/register/index.html) such a runner with your Gitlab instance if you haven't done so yet.
2. In `.gitlab-ci.yml` file in your client repo, add the following minimal configuration, replacing _indigo_ for your chosen distro:
```
image: docker:git
services:
  - docker:dind
before_script:
  - apk add --update bash coreutils tar
  - git clone https://github.com/ros-industrial/industrial_ci .industrial_ci
indigo:
  script: .industrial_ci/gitlab.sh ROS_DISTRO=indigo
```

Reply:
Thanks. I agree to just briefly mention the need for setting up Pipeline and rely on the external source for the detail. I've basically applied your suggestion in the updated commit, with a minor change to give a hint about unclarity of gitlab.com version and your-own-server version that I find not clear for the most of the times in the docs on Gitlab.